javascript-practice/src/js/views/registerView.js

```diff
@@ -23,7 +23,7 @@ export default class RegisterView extends CommonLoginRegisterView {
 
     const account = { email, password, passwordConfirm };
 
-    if (this.validateForm(account)) {
+    if (this.isValidateForm(account)) {
```

Comment:
this.isValidateForm(account) -> Should be isValidAccount

Reply:
Fixed.